corehq/apps/domain/static/domain/js/internal_subscription_management.js

```diff
@@ -0,0 +1,15 @@
+hqDefine('domain/js/internal_subscription_management', function() {
+    var viewModel = {
+        subscriptionType: ko.observable($('#id_subscription_type').val() || null),
+        trialLength: ko.observable(90),
+    };
+    viewModel.end_date = ko.computed(function() {
+        var date = new Date();
+        date.setHours(0, 0, 0, 0);
+        date.setDate(date.getDate() + parseInt(viewModel.trialLength()));
+        return date.toJSON().slice(0, 10);
+    });
+    $('#subscription_management').koApplyBindings(viewModel);
```

Comment:
Non-blocking: it'd be good to add a document ready handler that wraps the `koApplyBindings` and the datepicker initialization - shouldn't matter right now, because the js block is at the very end of `hqwebapp/base.html`, but still good practice.